_inc/client/components/dash-item/index.jsx

```diff
@@ -57,11 +63,20 @@ const DashItem = React.createClass( {
 					{ __( 'Upgrade' ) }
 				</Button>;
 				break;
+			case 'pro-uninstalled':
+				status = <Button
+					compact={ true }
+					primary={ true }
+					href={ 'https://wordpress.com/plugins/' + this.props.module + '/' + window.Initial_State.rawUrl }
+				>
+					{ __( 'Install' ) }
+				</Button>;
+				break;
 			case 'pro-inactive':
 				status = <Button
 					compact={ true }
```

Comment:
They are all aligned the same as of now.  Do you think they should be moved one way or another?